### PR TITLE
Feature/grouping payment

### PR DESCRIPTION
**Pull request by suryagsaputra**

### CODE REVIEW

File: corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/BankTransferCharge.java [hunk 1]
```diff
@@ -2,13 +2,11 @@
...
 import com.midtrans.sdk.corekit.core.snap.model.pay.request.CustomerDetailPayRequest;
-import com.midtrans.sdk.corekit.core.snap.model.pay.response.va.BcaPaymentResponse;
-import com.midtrans.sdk.corekit.core.snap.model.pay.response.va.BniPaymentResponse;
+import com.midtrans.sdk.corekit.core.snap.model.pay.response.BasePaymentResponse;
 import com.midtrans.sdk.corekit.core.snap.model.pay.response.va.OtherPaymentResponse;
-import com.midtrans.sdk.corekit.core.snap.model.pay.response.va.PermataPaymentResponse;
 import com.midtrans.sdk.corekit.utilities.Validation;
 
-public class MidtransBankTransfer extends BaseMidtransGroupPayments {
+public class BankTransferCharge extends BaseGroupPayments {
```
Comment (femmerling) [inline corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/BankTransferCharge.java]: Can we change `BaseGroupPayments` into `PaymentGroupBase` @suryagsaputra ?

File: corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/BaseGroupPayments.java [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import com.midtrans.sdk.corekit.core.MidtransSdk;
 import com.midtrans.sdk.corekit.core.snap.SnapApiManager;
 
-public class BaseMidtransGroupPayments {
+public class BaseGroupPayments {
```
Comment (femmerling) [inline corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/BaseGroupPayments.java]: same here

File: corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/CardlessCreditCharge.java [hunk 3]
```diff
@@ -4,7 +4,7 @@
 import com.midtrans.sdk.corekit.core.snap.model.pay.response.BasePaymentResponse;
 import com.midtrans.sdk.corekit.utilities.Validation;
 
-public class MidtransCardlessCredit extends BaseMidtransGroupPayments {
+public class CardlessCreditCharge extends BaseGroupPayments {
```
Comment (femmerling) [inline corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/CardlessCreditCharge.java]: same here

File: corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/EWalletCharge.java [hunk 4]
```diff
@@ -5,7 +5,7 @@
 import com.midtrans.sdk.corekit.core.snap.model.pay.response.BasePaymentResponse;
 import com.midtrans.sdk.corekit.utilities.Validation;
 
-public class MidtransEWallet extends BaseMidtransGroupPayments {
+public class EWalletCharge extends BaseGroupPayments {
```
Comment (femmerling) [inline corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/EWalletCharge.java]: and here

File: corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/OnlineDebitCharge.java [hunk 5]
```diff
@@ -2,11 +2,9 @@
 
 import com.midtrans.sdk.corekit.base.callback.MidtransCallback;
 import com.midtrans.sdk.corekit.core.snap.model.pay.response.BasePaymentResponse;
-import com.midtrans.sdk.corekit.core.snap.model.pay.response.bcaklikpay.BcaClickPayPaymentResponse;
-import com.midtrans.sdk.corekit.core.snap.model.pay.response.epaybri.BriEpayPaymentResponse;
 import com.midtrans.sdk.corekit.utilities.Validation;
 
-public class MidtransOnlineDebit extends BaseMidtransGroupPayments {
+public class OnlineDebitCharge extends BaseGroupPayments {
```
Comment (femmerling) [inline corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/OnlineDebitCharge.java]: here as well

File: corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/StoreCharge.java [hunk 6]
```diff
@@ -4,7 +4,7 @@
 import com.midtrans.sdk.corekit.core.snap.model.pay.response.BasePaymentResponse;
 import com.midtrans.sdk.corekit.utilities.Validation;
 
-public class MidtransStore extends BaseMidtransGroupPayments {
+public class StoreCharge extends BaseGroupPayments {
```
Comment (femmerling) [inline corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/StoreCharge.java]: of course here too

File: corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/BankTransferCharge.java (hunk 1 above)
Comment (suryagsaputra) [reply corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/BankTransferCharge.java]: Done

**Pull request merged**